test/unit/transaction.test.js

```diff
@@ -985,42 +985,33 @@ describe('Transaction', function() {
     it('should create payload when DT enabled and CAT disabled', function() {
       tx.agent.config.cross_application_tracer.enabled = false
 
-      const payload = tx.createDistributedTracePayload().text()
+      const payload = tx._createDistributedTracePayload().text()
 
       expect(payload).to.not.be.null
       expect(payload).to.not.equal('')
     })
 
-    it('generates a priority for entry-point transactions', () => {
-      expect(tx.priority).to.equal(null)
-      expect(tx.sampled).to.equal(null)
-
-      tx.createDistributedTracePayload()
-
-      expect(tx.priority).to.be.a('number')
-      expect(tx.sampled).to.be.a('boolean')
-    })
-
```

Comment:
This method should no longer generate priorities since it is not the entry-point. `insertDistributedTraceHeaders()` is and it will calculate priority.